avocado/core/job.py

```diff
@@ -89,7 +89,9 @@ def __init__(self, args=None):
         if unique_id is None:
             unique_id = job_id.create_unique_job_id()
         self.unique_id = unique_id
-        self.logdir = None
+        self._setup_job_results()
+        if not (self.standalone or getattr(self.args, "dry_run", False)):
+            self._update_latest_link()
```

Comment:
Actually, I noticed that `job.log` does exist, and uses an `avocado.app` based logger.  This reinforces the point that, a job can have logged messages that should be persisted.